Makefile

```diff
@@ -61,7 +61,15 @@ endif
 
 DATE    := $(shell date -u '+%Y-%m-%d %H:%M UTC')
 LDFLAGS := -ldflags='-w -X "main.Version=$(VERSION)" -X "main.BuildTime=$(DATE)"'
-GOFLAGS := CGO_ENABLED=0
+GOFLAGS := -v
+
+ifeq (,$(GOFLAGS))
+	ifeq (,$(findstring CGO_ENABLED=0,$(GO_ENVS)))
+		GO_ENVS := $(GO_ENVS) CGO_ENABLED=1
+	endif
+else
+	GO_ENVS := $(GO_ENVS) CGO_ENABLED=0
```

Comment:
I cannot make this work as "expected". Those variables are only expanded depending on how you run the command:

| Command                        | GO_ENVS                 |
| ------------------------------ | ----------------------- |
| `make build GO_ENVS="A=1 B=2"` | `A=1 B=2`               |
| `GO_ENVS="A=1 B=2" make build` | `A=1 B=2 CGO_ENABLED=0` |

So following the same convention:
| Command                              | GO_ENVS                      |
| ------------------------------------ | ---------------------------- |
| `GO_ENVS="CGO_ENABLED=1" make build` |`CGO_ENABLED=1 CGO_ENABLED=0` |

Reply:
Good catch, this is fixed now.